src/components/common/errorScreens/error.tsx

```diff
@@ -18,9 +19,7 @@ export class ErrorScreenManager extends Component<{
             case 401:
                 return 'Unauthorized'
             case 403:
-                return (
-                    <ErrorScreenNotAuthorized subtitle={this.props.subtitle} subtitleClass={this.props.subtitleClass} />
-                )
+                return <ErrorScreenNotAuthorized />
```

Comment:
what if want to apply different CSS as height of the header can change, so add an option to give class

Reply:
I think it is not required as just we are handling 403 here and class is present  in <ErrorScreenNotAuthorized />